drivers/storage/ebs/tests/test-env-down.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+# This script cleans up the infrastructure used for tests
+
+#set -e
+
+# Make sure that jq is installed
+hash curl 2>/dev/null || {
+  if [ -e "/etc/redhat-release" -o \
+         -e "/etc/redhat-version" ]; then
+    yum -y install curl
```

Comment:
In fact, why not just combine curl and jq in one install? will make the installs faster as well.

`yum -y install curl jq`
`apt-get install -y curl jq`
`brew install curl jq`